experimental/ag-ui/front-end/src/components/ChatAssistant.css

```
@@ -1,448 +1,529 @@
 
-.message-text blockquote {
-  border-left: 3px solid rgba(0, 0, 0, 0.2);
-  margin: 0.5rem 0;
-  padding-left: 0.75rem;
-  font-style: italic;
+/* ── Message Bubbles ── */
+.message {
+  max-width: 85%;
+  padding: 0.7rem 0.9rem;
+  border-radius: var(--radius-lg);
+  word-wrap: break-word;
```

Stylelint (17.7.0)

[error] Expected "word-wrap" to be "overflow-wrap" (property-no-deprecated)

(property-no-deprecated)